coreTable/OwidTable.ts

```diff
@@ -577,6 +588,72 @@ export class OwidTable extends CoreTable<OwidRow, OwidColumnDef> {
         )
     }
 
+    fillColumnWithTolerance(columnSlug: ColumnSlug, tolerance: number) {
+        const column = this.get(columnSlug)
+        const columnDef = column?.def as OwidColumnDef
+        const timeColumnSlug = timeColumnSlugFromColumnDef(columnDef)
+        const timeColumnDef = this.get(timeColumnSlug)?.def as OwidColumnDef
+        const originalTimeSlug = makeOriginalTimeSlugFromColumnSlug(columnSlug)
+
+        const originalRows = this.sortedByTime.rows
+        const allTimes = originalRows.map((row) => row[timeColumnSlug])
+
+        const rows = flatten(
```

Comment:
One idea: What about splitting this method into 2 transforms? Something like `tableWhereEveryEntityGetsATimeForAllTimes()` and then the Interpolate from closest as a separate transform on that?

Reply:
Should it be called `interpolateColumnWithTolerance`? My thinking is 1) we may add other (unrelated) interpolation functions in the future and might be helpful to find them. 2) In `grapher/stackedCharts/StackedUtils.ts` I actually think we could ditch those functions and then just fire up a CoreTable and reuse some of this code.

Reply:
Yeah that's a good idea! And easier to debug. 

I extracted `injectAllTimeRowsForEveryEntity()` as a separate transform. Feel free to rename it!